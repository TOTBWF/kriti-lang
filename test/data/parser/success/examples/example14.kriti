{ "foo": "{{obj['x-y']}}", "bar": "{{obj['a b cde-fgh']}}" }
