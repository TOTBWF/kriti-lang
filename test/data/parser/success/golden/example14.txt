Object
    ( fromList
        [
            ( "bar"
            , StringInterp
                ( SourcePosition
                    { _sourceName = "sourceName"
                    , _line = 1
                    , _column = 35
                    }
                , Just
                    ( SourcePosition
                        { _sourceName = "sourceName"
                        , _line = 1
                        , _column = 60
                        }
                    )
                )
                [ String ""
                , Path
                    [
                        (
                            ( SourcePosition
                                { _sourceName = "sourceName"
                                , _line = 1
                                , _column = 1
                                }
                            , Just
                                ( SourcePosition
                                    { _sourceName = "sourceName"
                                    , _line = 1
                                    , _column = 4
                                    }
                                )
                            )
                        , Obj "obj"
                        )
                    ,
                        (
                            ( SourcePosition
                                { _sourceName = "sourceName"
                                , _line = 1
                                , _column = 4
                                }
                            , Just
                                ( SourcePosition
                                    { _sourceName = "sourceName"
                                    , _line = 1
                                    , _column = 15
                                    }
                                )
                            )
                        , Obj "a b cde-fgh"
                        )
                    ]
                ]
            )
        ,
            ( "foo"
            , StringInterp
                ( SourcePosition
                    { _sourceName = "sourceName"
                    , _line = 1
                    , _column = 10
                    }
                , Just
                    ( SourcePosition
                        { _sourceName = "sourceName"
                        , _line = 1
                        , _column = 26
                        }
                    )
                )
                [ String ""
                , Path
                    [
                        (
                            ( SourcePosition
                                { _sourceName = "sourceName"
                                , _line = 1
                                , _column = 1
                                }
                            , Just
                                ( SourcePosition
                                    { _sourceName = "sourceName"
                                    , _line = 1
                                    , _column = 4
                                    }
                                )
                            )
                        , Obj "obj"
                        )
                    ,
                        (
                            ( SourcePosition
                                { _sourceName = "sourceName"
                                , _line = 1
                                , _column = 4
                                }
                            , Just
                                ( SourcePosition
                                    { _sourceName = "sourceName"
                                    , _line = 1
                                    , _column = 7
                                    }
                                )
                            )
                        , Obj "x-y"
                        )
                    ]
                ]
            )
        ]
    )
